news/2023/2023-11-16-osu-world-cup-2023-quarterfinals-recap.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "osu! World Cup 2023: Quarterfinals Recap"
+date: 2023-11-16 12:00:00 +0000
+---
+
+Tune in to catch up on the latest upsets and heartbreaks (of which there were many), and prepare yourself for the upcoming semifinals weekend.
+
+![](/wiki/Tournaments/OWC/2023/img/owc2023-banner.jpg)
+
+Do you remember the crazy quarterfinals stage of the [2021 osu! World Cup](/wiki/Tournaments/OWC/2021#quarterfinals.1), including five tiebreakers? Not as many were needed the past weekend, but the amount of upsets and close battles made up for it and delivered one of the most entertaining stages in some years.
+
+As so many highlights needed to be included, the recap video is longer than usual. In case you just need a quick update, look below for the most outstanding matches.
```

Comment:
```suggestion
Because so many exciting moments would've each constituted a crime not to highlight, the recap video is longer than usual. In case you just need a quick update, read on below for the most outstanding matches.
```